test/support/builders/atom_watcher_event.js

```diff
@@ -96,13 +87,12 @@ module.exports = class AtomWatcherEventBuilder {
   }
 
   ino (newIno /*: number */) /*: this */ {
-    if (this._event.stats == null) this._event.stats = buildStats(this._event.kind)
-    this._event.stats.ino = newIno
+    this._ensureStatsBuilder().ino(newIno)
     return this
   }
 
   noStats () /*: this */ {
-    delete this._event.stats
```

Comment:
We should keep this line. If we build an event atop another one that has stats, the `stats` attribute will be set.